search-sentiments-map/routes/sentiment.js

```diff
@@ -62,4 +46,5 @@ async function getSentimentScore(searchResultTitleSnippet) {
   }
 }
 
-module.exports.router = router;
\ No newline at end of file
+module.exports.router = router;
```

Comment:
If we no longer use router here, we can delete this line and delete line 17 too.

Reply:
Similarly, delete sentiment router from the app.js file.